### PR TITLE
normalizing frames with custom mean_std instead of defaulting to resnet

### PR DESCRIPTION
**Pull request by Dee61298**
* ***Replaced norm_resnet by norm_meanstd*** : 
You can now normalize frame with custom mean_std instead of defaulting to resnet.
Either you pass the mean_std at instanciation or you pass it as a parameter to the norm_meanstd function : 
```python
import torch
import aloscene

x=torch.rand(3,600,600)
x=aloscene.Frame(x,mean_std=((0.333,0.333,0.333),(0.333,0.333,0.333)))

x=x.norm_meanstd()
print("normalization de x ",x.normalization)
print("Mean_std de x",x.mean_std)

x=x.norm_meanstd(name="resnet")
print("normalization de x ",x.normalization)
print("Mean_std de x",x.mean_std)

x=x.norm_meanstd(mean_std=((0.440,0.220,0.880),(0.333,0.333,0.333)), name="my_norm")
print("normalization de x ",x.normalization)
print("Mean_std de x",x.mean_std)
```

* ***Output*** : 
```python
normalization de x  None
Mean_std de x ((0.333, 0.333, 0.333), (0.333, 0.333, 0.333))
normalization de x  resnet
Mean_std de x ((0.485, 0.456, 0.406), (0.229, 0.224, 0.225))
normalization de x  my_norm
Mean_std de x ((0.44, 0.22, 0.88), (0.333, 0.333, 0.333))

```

_____
This pull request includes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: aloscene/frame.py [hunk 1]
```diff
@@ -507,10 +510,27 @@ def mean_std_norm(self, mean, std, name) -> Frame:
 
         return tensor
 
+    def norm_meanstd(self, mean_std=None, name=None) -> Frame:
```
Comment (Data-Iab) [inline aloscene/frame.py:513]: Having two methodes with nearly the same name is not a good practice. try to fix `mean_std_norm` instead

File: aloscene/frame.py [hunk 2]
```diff
@@ -507,10 +510,27 @@ def mean_std_norm(self, mean, std, name) -> Frame:
...
+        Examples
+        --------
+        >>> frame_resnet = frame.norm_resnet(name="resnet")
+        >>> frame_custom_norm = frame.norm_resnet(name="custom")
+        """
+        if name == "resnet":
+            return self.norm_resnet()
+        elif mean_std is not None and len(mean_std) == 2:
+            return self.mean_std_norm(mean=mean_std[0], std=mean_std[1], name=name)
+        elif self.mean_std is not None:
```
Comment (Data-Iab) [inline aloscene/frame.py:526]: there's no difference between this condition and the one above.

File: aloscene/frame.py [hunk 3]
```diff
@@ -507,10 +510,27 @@ def mean_std_norm(self, mean, std, name) -> Frame:
...
+            return self.mean_std_norm(mean=self.mean_std[0], std=self.mean_std[1], name=name)
```
Comment (Data-Iab) [inline aloscene/frame.py:527]: The attribute `mean_std` is used to store the current normalization parameters. You can delete this

**Review by Data-Iab — CHANGES_REQUESTED** [review]

**Comment by Data-Iab** [comment]
Thing you can add : normlization from minmax -> meanstd
```python
import torch
import aloscene

x = torch.rand(3,10,10)
x = aloscene.Frame(x,mean_std=((0.333,0.333,0.333),(0.333,0.333,0.333)))

x = x.norm_minmax_sym()
x = x.mean_std_norm(mean=(0.333,0.333,0.333), std=(0.333,0.333,0.333), name="custom")  # Exception raised
```

**Comment by Dee61298** [comment]
Duplicated in #317 with the correct commits. Closing this one.
